WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/PromptProgressBar/ProductCreationAIPromptProgressBar.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+struct ProductCreationAIPromptProgressBar: View {
+    @Binding var text: String
+    @StateObject private var viewModel = ProductCreationAIPromptProgressBarViewModel()
+
+    var body: some View {
+        VStack(content: {
+            ProgressView(value: viewModel.status.progress)
+                .progressViewStyle(ProductCreationAIBarProgressStyle(color: viewModel.status.color))
+
+
+            HStack(spacing: 0, content: {
+                Text(viewModel.status.mainDescription)
+                    .font(.footnote)
+                    .fontWeight(.semibold) +
+                Text(viewModel.status.secondaryDescription)
+                    .font(.footnote)
+                    .fontWeight(.regular)
+
+                Spacer()
+            })
+            .animation(.easeIn, value: viewModel.status)
+            .padding(.top, Layout.padding/2)
```

Comment:
1. For simplicity, we should specify the spacing in VStack instead of using top padding.
2. The spacing should be `8` instead of `16`.

Reply:
@selanthiraiyan I strongly followed the padding of the Figma design, where the padding it's 16 from borders, and 8 between the progress bar and the prompt suggestion. 🤔

Reply:
Here the `padding` modifier with the `top` parameter determines the vertical spacing between the progress view and the text. As per the Figma design, the spacing should be `8`, and currently it is `16`.

<img src="https://github.com/woocommerce/woocommerce-ios/assets/524475/a4ba652b-4a85-4f16-871d-bef4afad36dc" width="320"/>

Reply:
Oh got it. Fixed here 13ea31ad99a745616f85cccafce458d055f3decf